WooCommerce/Classes/ViewRelated/Products/Cells/ProductsTabProductTableViewCell.swift

```diff
@@ -118,6 +126,9 @@ private extension ProductsTabProductTableViewCell {
         contentView.addSubview(bottomBorderView)
         contentView.pinSubviewToAllEdges(stackView, insets: UIEdgeInsets(top: 16, left: 16, bottom: 16, right: 16))
 
+        // Not initially enabled, saved for possible compact icon case
+        productImageViewFixedHeightConstraint = productImageView.heightAnchor.constraint(equalTo: stackView.heightAnchor)
```

Comment:
nit: how about moving it to `configureProductImageView` next to the `productImageViewRelationalWidthConstraint` configuration?

Reply:
Unfortunately stack views are not exposed out of `configureSubviews` and pinning height of `productImageView` to the main `contentView` produces strange broken layout.